### PR TITLE
Add docs for filter config and java client controller url init

### PR DESCRIPTION
**Pull request by saurabhd336**
Add docs for 
1) Filter config support for SQL-like expressions with AND / OR / NOT (https://github.com/apache/pinot/pull/8582)
2) Java client controller URL init (https://github.com/apache/pinot/pull/8467)

### CODE REVIEW

**Comment by saurabhd336** [comment]
@KKcorps

**Comment by KKcorps** [comment]
Can we also add section on transform and filter configs here -
https://docs.pinot.apache.org/configuration-reference/table

File: developers/advanced/ingestion-level-transformations.md [hunk 1]
```diff
@@ -143,6 +143,16 @@ Consider a table that has a string column `campaign` and a multi-value column do
 }
 ```
 
+Filter config also supports SQL-like expression of inbuilt functions for filtering records (starting v 0.11.0+).
```
Comment (navina) [inline developers/advanced/ingestion-level-transformations.md]: qq: I understand `SQL-like` expression. But what inbuilt functions are you referring to here? perhaps add a link to it?

**Review by navina — APPROVED** [review]
just one question. otherwise, lgtm!

File: developers/advanced/ingestion-level-transformations.md (hunk 1 above)
Comment (KKcorps) [reply developers/advanced/ingestion-level-transformations.md]: +1. Add link to the following page - https://docs.pinot.apache.org/configuration-reference/functions

Comment (saurabhd336) [reply developers/advanced/ingestion-level-transformations.md]: Ack. Added the link to scalar functions page.

**Comment by saurabhd336** [comment]
@KKcorps https://docs.pinot.apache.org/configuration-reference/table already has section for ingestionConfig that links to this page. That page seems to only talk about top level table config fields.

**Review by KKcorps — APPROVED** [review]
LGTM! Link to this page already exists on the table config page as mentioned by Saurabh

**Pull request merged**